### PR TITLE
gp: remove leftovers from GP test suite v1.1.0.4 support

### PR DESCRIPTION
**Pull request by jforissier**
Since f7c37e7c29eb ("gp: update compliance suite to version 2.0.0.2") it
is not possible to build the GlobalPlatform TEE Initial Configuration
Test Suite v1.1.0.4. The test suite was upgraded and the build system
switched from GNU make to CMake.

This commit removes some leftovers in the Makefiles from the previous
configuration.

Signed-off-by: Jerome Forissier <jerome@forissier.org>

### CODE REVIEW

**Comment by jforissier** [comment]
@jenswi-linaro am I missing something here?

File: host/xtest/Makefile [hunk 1]
```diff
@@ -100,16 +100,6 @@ CFLAGS += -I$(OPTEE_PKCS11_TA_HEADERS)
...
-ifdef CFG_GP_PACKAGE_PATH
-CFLAGS += -DWITH_GP_TESTS
-
-srcs  += gp_7000.c \
-	 gp_7500.c \
-	 gp_8000.c \
-	 gp_8500.c \
-	 gp_9000.c
-endif
-
```
Comment (jenswi-linaro) [inline host/xtest/Makefile:112]: The other `ifdef CFG_GP_PACKAGE_PATH` and the `ifndef CFG_GP_PACKAGE_PATH` below can also be removed.

Comment (jforissier) [reply host/xtest/Makefile:112]: How could I miss them...? Fixed now.

**Comment by jenswi-linaro** [comment]
Thanks for cleaning up after me. :-)

`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**